src/go/rdctl/pkg/factoryreset/delete_data.go

```diff
@@ -34,6 +34,29 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func addAppHomeWithoutSnapshots(appHome string) []string {
```

Comment:
This function name makes sense on macOS, but doesn't make sense on Linux because these directories are not in `Paths.AppHome`.

Also, the `appHome` argument/variable should have a different name.